cmd/image.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/cluster-version-operator/pkg/cvo"
+)
+
+var (
+	imageCmd = &cobra.Command{
+		Use:     "image",
+		Short:   "Returns image for requested short-name from UpdatePayload",
+		Long:    "",
+		Example: "%[1] image <short-name>",
```

Comment:
Does the `%[1]` prints the parent command?

Reply:
> Does the %[1] print the parent command?

No, will fix that in follow up PR.